docker-storage-setup.spec

```diff
@@ -30,17 +24,7 @@ as the root logical volume and partition table.
 %build
 
 %install
-install -d %{buildroot}%{_bindir}
-install -p -m 755 %{SOURCE0} %{buildroot}%{_bindir}/docker-storage-setup
-install -d %{buildroot}%{_unitdir}
-install -p -m 644 %{SOURCE1} %{buildroot}%{_unitdir}
```

Comment:
If we are getting rid of macros like _unitdir, are we not going to be broken if tomorrow systemd changes location of unit files and this macro starts expanding to a different value.